### PR TITLE
mesapy: support file operation functions in the python payload

### LINKED ISSUE #8: [WIP] Change interface between FNS and Worker
Current interface between FNS and Worker needs to be changed.
1. Provide global functions for data reading/writing
    To support mesapy and splited workers, global functions are needed. Then the worker can directly read/write data with context-id/context-token. 
2. A structure for Running Task. 
    The structure will hold the task information, store results and enforce some security policies. 
    The instances of the structure will be saved to the global area. So the above global functions can retrieve the instance and then read/write data. 
3.  A structure for Worker Info and a structure for Worker
4. Worker queue
      The FNS should maintain the available workers

### PR DESCRIPTION
**Pull request by mssun**
## Description

Support file operation functions in the MesaPy worker. Usage can be seen in example payload: `examples/py_file/payload.py`.

Fixes #8

## Type of change (select applied and DELETE the others)

- [x] New feature (non-breaking change which adds functionality)
- [ ] API change with a documentation update

## How Has This Been Tested?

By the `py_file` project in examples.

## Checklist (check ALL before submitting PR, even not applicable)

- [x] Fork the repo and create your branch from `master`.
- [x] If you've added code that should be tested, add tests.
- [x] If you've changed APIs, update the documentation.
- [x] Ensure the tests pass (see CI results).
- [x] Make sure your code lints/format.

### CODE REVIEW

**Comment by mssun** [comment]
Seems that there is no place to document the usages/functionalities of our builtin workers. I'll not write the doc for these file related functions *for now*. I think the `payload.py` file (`examples/py_file/payload.py`) in `py_file` example is self-explained.

File: examples/py_file/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+use lazy_static::lazy_static;
+use mesatee_sdk::{Mesatee, MesateeEnclaveInfo};
+use std::net::SocketAddr;
+use std::{env, fs};
+
+static FUNCTION_NAME: &str = "mesapy_from_buffer";
+static USER_ID: &str = "uid";
+static USER_TOKEN: &str = "token";
+
+lazy_static! {
```
Comment (dingelish) [inline examples/py_file/src/main.rs]: should we use mesatee_config here? i'm not sure

Comment (mssun) [reply examples/py_file/src/main.rs]: No, `mesatee_config` is for `core`, `services`, etc. This is an example, hard-coded addresses are intuitive for others reading the code. Other examples are also using the same definitions here.

File: examples/py_file/payload.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+def save_file_for_all_participants(context_id, context_token):
+    content = "save_file_for_all_participants"
+    saved_id = mesatee.mesatee_save_file_for_all_participants(context_id, context_token, content)
+    content_from_file = mesatee.mesatee_read_file(context_id, context_token, saved_id)
+    if content_from_file == content: return True
+    else: return False
+
+def save_file_for_file_owner(context_id, context_token, file_id):
+    print file_id
```
Comment (litongxin1991) [inline examples/py_file/payload.py]: Should we remove "print" in python script?

**Pull request merged**